### PR TITLE
Add label in CONTRIBUTING.md, fix broken link

### LINKED ISSUE #1127: Add the new labels' details in the CONTRIBUTING.md file
### Summary

* I have created a new label "[good first PR review](https://github.com/alan-turing-institute/the-turing-way/pulls?q=is%3Apr+is%3Aopen+label%3A%22good+first+PR+review%22)" for new contributors to get started with reviewing in this repo

### What needs to be done?

- [ ] The label is not yet added in the [CONTRIBUTING.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) which can be added in the [issue label section](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md#issue-labels) with a one-sentence explanation (see [here for reference](https://github.com/alan-turing-institute/the-turing-way/issues/labels))

### Who can help?

* Anyone who is willing to add this in the designated area as described above
* preferably new users who have never contributed to this repo

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by srishti-nema**
### Summary

*Fixes #1127 . Fixed [this](https://github.com/alan-turing-institute/the-turing-way/blob/master/.github/ISSUE_TEMPLATE.md) link.*

### List of changes proposed in this PR (pull-request)

* *Added the label for [good first PR review](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20PR%20review) in [CONTRIBUTING.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)*
* *Added [issue template](https://github.com/alan-turing-institute/the-turing-way/blob/master/.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md)  link in [this](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md#1-comment-on-an-existing-issue-or-open-a-new-issue-referencing-your-addition) section.*

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -171,6 +171,10 @@ The list of labels for current issues can be found [here][turing-way-labels] and
 
 - [![outreach](https://img.shields.io/badge/-Outreach-fcbae8.svg)][labels-outreach] _These issues relate to topics to reach out to the community._
 
+- [![good-first-PR-review](https://img.shields.io/badge/-good%20first%20PR%20review-C992E0.svg)](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20PR%20reviews) _These pull requests are for the new maintainers of The Turing Way to start with reviewing and merging tasks_
```
Comment (malvikasharan) [inline CONTRIBUTING.md:174]:
```suggestion
- [![good-first-PR-review](https://img.shields.io/badge/-good%20first%20PR%20review-C992E0.svg)] These pull requests are for the new members of The Turing Way community who want to start with reviewing and approving some simple pull requests._
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -171,6 +171,10 @@ The list of labels for current issues can be found [here][turing-way-labels] and
...
+
+  If you are a new maintainer of The Turing way and are anxious about how to start reviewing and merging various contributions, these pull requests act as a great starting point for you to begin. Issues like small modifications, typo errors and minor bug fixes are resolved by these PRs which are easy to review as a beginner.
```
Comment (malvikasharan) [inline CONTRIBUTING.md:176]:
```suggestion
  If you are a new member of The Turing way and are looking for opportunities to start as a reviewer of contributions made on our Github repository, these pull requests are a great starting point for you. Issues like small modifications, typo errors and minor bug fixes are resolved by these PRs which are easy to review as a beginner.
```

**Comment by malvikasharan** [comment]
@all-contributors please add @srishti-nema for bug fixes.

File: CONTRIBUTING.md (hunk 1 above)
Comment (srishti-nema) [reply CONTRIBUTING.md:174]: Does clicking on the label redirect to [this](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20PR%20review) link? I tried to refer to the format as given for labels but couldn't implement the redirection part, hence added the link.

File: CONTRIBUTING.md (hunk 2 above)
Comment (srishti-nema) [reply CONTRIBUTING.md]: Looks better! I will try to improve and become more clear in future. Thank you!

File: CONTRIBUTING.md (hunk 1 above)
Comment (malvikasharan) [reply CONTRIBUTING.md:174]: That format works for other labels, this should work as well.

Comment (malvikasharan) [reply CONTRIBUTING.md:174]: Oh, and in suggestions made with a comment this the original comment on this thread, you can directly add then by clicking on "commit suggestion" so you don't need to re-edit them in.

Comment (srishti-nema) [reply CONTRIBUTING.md:174]: Yes, did that for the other change. Will do that here as well.

**Comment by malvikasharan** [comment]
Thanks @srishti-nema! If you see the merge pr button, go ahead and merge it! 🎉

**Comment by srishti-nema** [comment]
> Thanks @srishti-nema! If you see the merge pr button, go ahead and merge it! tada

I can't merge it, I think.

**Comment by srishti-nema** [comment]
Hey @malvikasharan , I just noticed that the label is not directing to the correct link and the text is not italicized for some reason. This is just some formatting problem I think.

**Pull request merged**